kaitaistruct.lua

```diff
@@ -1,5 +1,14 @@
 local class = require("class")
 local stringstream = require("string_stream")
+local lunpack = require("struct").unpack
+
+local function try_require(name)
+    local success, mod = pcall(require, name)
+    if success then return mod else return nil end
```

Comment:
I'd prefer to spread this `if..else` statement across multiple lines, because it's more legible and understandable:

```suggestion
    if success then
        return mod
    else
        return nil
    end
```

I think there's just too many lowercase words on one line. It's not so bad when the syntax highlighting works, but consider this:

```
    if success then return mod else return nil end
```

This looks rather like a list of keywords than a line of code.

Reply:
Agreed